### ISSUE TITLE
Different mouse profiles  [enhancement]

### ISSUE THREAD

**Issue by NikEfth** [noah-nuebling/Mac-Mouse-Fix#488]
## Description
Hi there, 
I would like first to say thank you. This is one of the best utilities for macOS ever. 

I would like to ask for different profiles, if possible. 

## Rationale
I have different mice in my office and home. 
I have a regularly shaped mouse at my office, but a trackball at home. 
So the same shortcuts don't work well with both types. 

## Additional info
–

**Comment by flightmansam** [comment]
Agreed! I have a MxMaster 3 that has a very heavy scroll wheel (inbuilt inertia) and a crappy mouse at work (which could really benefit from Mac Mouse Fix 3). I don't want to have to manually toggle it all the time.
